alexa/request/certificate.php

```diff
@@ -41,6 +41,10 @@ public function __construct( $certificate_url, $signature, $app_id = '' ) {
 	}
 
 	public function validate_request( $request_data ) {
+
+		// Set required http status code 400 for certificate error
+		http_response_code( 400 );
```

Comment:
Sure. I only meant to mention it in light of the other issue; I defer to you about the minimum PHP version.